build/yamls/antrea-windows-with-ovs.yml

```diff
@@ -54,6 +54,27 @@ data:
       netcfg -l $OVSDriverDir/ovsext.inf -c s -i OVSExt
     }
 
+    # Check if redist is already installed
```

Comment:
I probably just don't know it, but is "redist" a standard term?

Reply:
No, it is to express [Visual C++ Redistributable](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022). As the files in the official download link is named as vc_redist.xxx.exe, so we use "redist" as the directory name. It is consistent with the directory name in nsx-ovs deliverables.

Reply:
ok, it sounds like vs_redist would be a bit more accurate, I have seen it used in other places in the code